src/ildasm/exe/CMakeLists.txt

```diff
@@ -35,6 +35,20 @@ set(ILDASM_SOURCES
     ../windasm.cpp
 )
 
+if (WIN32)
```

Comment:
I've mentioned this one too in my previous comment, but I guess I should have added a separate comment.

Reply:
Fixing